src/main/java/io/prometheus/cloudwatch/CloudWatchCollector.java

```diff
@@ -168,12 +173,22 @@ private CloudWatchCollector(JSONObject config, AmazonCloudWatchClient client) th
     }
 
     /**
-     * Check if a metric should be used according to `aws_dimension_select`
+     * Check if a metric should be used according to `aws_dimension_select` or `aws_dimension_select_regex`
      */
     private boolean useMetric(MetricRule rule, Metric metric) {
-      if (rule.awsDimensionSelect == null) {
+      if (rule.awsDimensionSelect == null && rule.awsDimensionSelectRegex == null) {
+        return true;
+      }
+      if (metricsIsInAwsDimensionSelect(rule, metric) || metricIsInAwsDimensionSelectRegex(rule, metric)) {
```

Comment:
I changed this.
Now it is forbidden to  set `aws_dimension_select` and `aws_dimension_select_regex` at the same time:
https://github.com/Jimdo/cloudwatch_exporter/blob/dimension_value_wildcard/src/main/java/io/prometheus/cloudwatch/CloudWatchCollector.java#L110